language/en/units.json

```diff
@@ -1714,7 +1714,7 @@
 			"legavrad": "Radar Vehicle",
 			"legshot": "Shielded Riot Defence Bot",
 			"legsrail": "All-Terrain Heavy Railgun",
-			"legsrailt4": "All-Terrain Heavy Railgun",
+			"legsrailt4": "All-Terrain Amphibious Heavy Sniper Railgun",
```

Comment:
Does this even fit on the UI?

Reply:
Easily.
![obraz](https://github.com/user-attachments/assets/c380b0cc-27a3-4320-92c1-e13ef645b24f)